RestClients.cs

```diff
@@ -0,0 +1,92 @@
+﻿using APIAutomation;
+using NUnit.Framework;
+using RestSharp;
+
+namespace APIAutomation
+{
+    public class ClientForWriteScope
+    {
+        private static ClientForWriteScope _instance;
+        private static readonly object _lock = new object();
+        private readonly RestClient _client;
+
+        private ClientForWriteScope(string baseURL, string clientUsername, string clientPassword)
+        {
+            var options = new RestClientOptions(baseURL)
+            {
+                Authenticator = new BasicAuthenticator(baseURL, clientUsername, clientPassword, "write")
+            };
+
+            _client = new RestClient(options);
+        }
+
+        public static ClientForWriteScope GetInstance(string baseURL, string clientUsername, string clientPassword)
```

Comment:
It's not critical point in this case, but I would recommend you to use GetInstance() method without using any parameters, because you will need to them all each time when the instance from this class will be returned. It will be enough to put these parameters into .runsettings file or something like that. I guess that this refactoring can be implemented in future.

Reply:
Ok, it will be done in task 40